runtime/manifest.js

```diff
@@ -590,6 +590,7 @@ ${e.message}
     // TODO: annotate other things too
     let recipe = manifest._newRecipe(recipeItem.name);
     recipe.annotation = recipeItem.annotation;
+    recipe.tags = recipeItem.tags;
```

Comment:
We need to better align this with particles, where tags == verbs

Reply:
As in, we should start to call this `recipe.verbs` and migrate the `lowerCase` recipe names to be `#tagformat`

Reply:
I would like us to maintain both the lowerCase name and the #tagformat verbs, please.

Reply:
I clarified with Shane, we should support both (uppercase!) `Name` and `#verb` for both particle and recipe.

Reply:
done.
the recipe name currently allows both - upper and lower case. should i remove the lower case one?